docs/operate-and-deploy/installation/upgrading.md

```diff
@@ -615,4 +615,26 @@ SELECT * FROM input PARTITION BY col0 EMIT CHANGES;
 ### `ALL`, `WINDOWSTART` and `WINDOWEND` are now reserved identifiers
 
 Any query using these identifiers will need to be changed to either use some other identifier, or
-to quote them.
\ No newline at end of file
+to quote them.
+
+
+
+### Upgrading from confluentksqlDB 5.5 to confluentksqlDB 6
```

Comment:
These details for in-place upgrades are applicable for all in-place upgrades, not just from 5.5 to 6.0. How about rewriting them to be generic from `<current_version>` to `<new_version>` and moving this section to be part of the section titled "How to upgrade" above? We can have two sub-sections under "How to upgrade": "In-place upgrades" (which you've provided the content for) and "Upgrades that cannot be performed in place" (the existing content under that section).

Reply:
Understood and changes done in my branch.